website_form_builder/static/src/css/website_form_builder.less

```
@@ -15,10 +15,12 @@
     }
 }
 
-body .modal.o_website_modal {
+/* Make modals with multiselect with a usable height */
+body .modal.o_technical_modal {
     textarea, select[multiple] {
         &.form-control {
             height: auto;
+            background: None;
```

Let's be more specific to allow theming things such as bg color, etc. Also standard CSS is lowercase always:

```suggestion
            background-image: none;
```